x-pack/plugin/eql/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/eql/10_basic.yml

```diff
@@ -49,6 +59,55 @@ setup:
   - match: {hits.events.1._id: "2"}
   - match: {hits.events.2._id: "3"}
 
+---
+"Execute EQL events query with fields filtering":
+  - do:
```

Comment:
should those have maybe the:
```
- skip:
      version: " - 7.12.99"
```
block?

Reply:
I guess the 7.12 port will have the versions kept as in original PR?

Reply:
There won't be a 7.12 port, only 7.x.

Reply:
I don't think the EQL yaml tests run in a mixed cluster (or any other bwc context) scenario that warrants for this `skip` block.